### PR TITLE
feat(global-metrics): use pure kad exporter for a lightweight public dashboard

### PR DESCRIPTION
**Pull request by ozwaldorf**
## Why

Our existing global metrics setup is scraping individual nodes, and is extremely heavy. 

## What

- refactor `docker/global-metrics` around kademlia exporter

## Demo

deployed at https://dashboard.ursa.earth/

## Checklist

- [x] ~~I have made corresponding changes to the tests~~
- [x] ~~I have made corresponding changes to the documentation~~
- [x] I have run the app using my feature and ensured that no functionality is broken

### CODE REVIEW

File: docker/global-metrics/data/exporter.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+max_mind_db_path = "/var/lib/GeoIP/GeoLite2-City.mmdb"
+# cloud_provider_cidr_db_path = "<path to cidrs.csv>"
```
Comment (b0xtch) [inline docker/global-metrics/data/exporter.toml:2]: dead

File: docker/global-metrics/data/prom/prometheus.yml [hunk 2]
```diff
@@ -2,18 +2,16 @@ global:
...
-    http_sd_configs:
-      - url: http://ursa-tracker:4000/http_sd
-        refresh_interval: 5s
-
   - job_name: 'bootstrap'
     static_configs:
       - targets: [
-        'bootstrap-node-1.ursa.earth:4070',
-        'bootstrap-node-2.ursa.earth:4070',
+        'bootstrap0.ursa.earth',
```
[the remark this answers is not in the document]
Comment (ozwaldorf) [reply docker/global-metrics/data/prom/prometheus.yml:8]: just to note the previous dns addresses were never setup, but I did setup bootstrap0 recently. Will update the dns in a bit once I get into the casa

[the remark this answers is not in the document]
Comment (ozwaldorf) [reply docker/global-metrics/data/prom/prometheus.yml]: https://github.com/fleek-network/ursa/pull/406/commits/dce3db183573380f18d090b9c6fe70bd97f2d97d

Will provide bootstrap metrics on the global dashboard based on these scrapes in a future PR

File: docker/global-metrics/data/exporter.toml (hunk 1 above)
Comment (ozwaldorf) [reply docker/global-metrics/data/exporter.toml]: https://github.com/fleek-network/ursa/pull/406/commits/9868e2b000066fcb5760ac7c138283df54de70cb

**Comment by b0xtch** [comment]
are we going to maintain these `.json` dashboards? How often do you think they will change?

**Comment by ozwaldorf** [comment]
Deployed at https://dashboard.ursa.earth/

**Comment by mxinden** [comment]
Great to see parts of the Kademlia exporter reused here. Let me know in case I can be of any help.

> They're based off of `kademlia-exporter.max-inden.de`, of which I couldn't find a git version uploaded. I had to make some minor modifications as well.

I don't have a public repository for those. Sorry. I am in favor of maintaining the rust-libp2p dashboard on the rust-libp2p monorepo, though I haven't gotten around to a pull request yet. Help more than welcome.

File: docker/global-metrics/data/exporter.toml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+max_mind_db_path = "/var/lib/GeoIP/GeoLite2-City.mmdb"
+
+name = "ursa"
+protocol_name = "/ursa/kad/0.0.1"
+bootnodes = [
+  "/dns/bootstrap-node-0.ursa.earth/tcp/6009/p2p/12D3KooWDji7xMLia6GAsyr4oiEFD2dd3zSryqNhfxU3Grzs1r9p"
+]
```
Comment (mxinden) [inline docker/global-metrics/data/exporter.toml:7]: Also happy to deploy and maintain a Kademlia Exporter instance on kademlia-exporter.max-inden.de/ scraping the ursa network. It is just a small configuration change on my end. Is the above bootstrap node address stable?

**Review by mxinden — COMMENTED** [review]
:rocket:

File: docker/global-metrics/data/exporter.toml (hunk 3 above)
Comment (ozwaldorf) [reply docker/global-metrics/data/exporter.toml:7]: Would be awesome to see! Thanks for the comments. And yeah, the address above is stable if you do run your own

**Review by b0xtch — APPROVED** [review]

**Comment by mxinden** [comment]
kademlia-exporter.max-inden.de is now as well monitoring the ursa network. See:

https://kademlia-exporter.max-inden.de/d/Pfr0Fj6Mk/rust-libp2p?orgId=1&refresh=30s&var-data_source=Prometheus&var-instance=kademlia-exporter-ursa:8080

https://kademlia-exporter.max-inden.de/d/-avwMhsik/kademlia-exporter?orgId=1&var-instance=kademlia-exporter-ursa:8080

Let me know if this is of any help.

**Pull request merged**